web.go

```diff
@@ -71,6 +71,9 @@ func Recover(handler http.Handler) http.Handler {
 }
 
 // LogRequest is a middleware that logs a request
+// HTTP status < 400 will be logged as Info
+// HTTP status <=400 <500 will be logged as Info with the body as message
```

Comment:
Don't we mean >=400 <500 ? Or in plain English, from 400 (inclusive) to 500 (exclusive)